modules/ROOT/pages/websockets/websocket-connector-configuration.adoc

```diff
@@ -0,0 +1,75 @@
+= About Websocket Connection Configuration
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket
+
+== About Websocket Connector Configuration
+
+There are two possible approaches to configuring a Websocket Connector. Both approaches will require an underlying HTTP Configuration from which several connection settings will be inherited, however these settings can be overridden in the Websocket configuration or at the operation level.
+
+=== Approach 1: Mimic HTTP Connector
```

Comment:
This entire page is wrong. This was the spec decision process on why to pick one approach other the other. Only approach 2 got implemented.

This page should explain approach 2 in depth. Giving examples, explaining the parameters. For example:

* Explain that you can choose to only provide server settings if your app only accepts connections but doesn't open any
* Explain that you can choose to only provide client settings if your app only creates outbound connections but doesn't listen to incoming ones
* You can provide both if your app does both things
* Explain why you need to reference an http:listener-config in the server part, and how would you use that to configure TLS and other http related things
* explain how the default host, port, path parameter of the client settings section work and why are they optional and you might choose not to use them at all.
* Etc, etc, etc